adapter-base/src/test/resources/ocsp/cacert.pem

```diff
@@ -0,0 +1,12 @@
+-----BEGIN CERTIFICATE-----
+MIIBtTCCATugAwIBAgIBADAKBggqhkjOPQQDBDAaMRgwFgYDVQQDDA9pb3QuZWNs
```

Comment:
will any of the tests fail if any of the ceriticates' validity period has expired?

Reply:
Yeas it will, but validity of new certificates is set to 50 years so I hope it is sufficient countermeasure.